src/Middleware/RequestThrottling/src/RequestThrottlingMiddleware.cs

```diff
@@ -20,8 +20,6 @@ public class RequestThrottlingMiddleware
         private readonly RequestDelegate _onRejected;
         private readonly ILogger _logger;
 
-        private int _queuedRequests;
```

Comment:
As we chatted about earlier, I don't know if we should rely on EventSource to count the number of requests for the QueuedRequestCount. It looks like you need to instantiate the _queueCount to zero, which seems odd too.

Reply:
Expose the current queue length through an internal field in `IQueuePolicy`? The field would make testing better, but would be internal and only used in testing.

Reply:
Maybe make a `TestQueuePolicy` which implements `IQueuePolicy` but has additional fields. Use this test policy in the tests. This is the "correct" approach but adds code and abstraction in the tests.

Reply:
Already had a `TestQueuePolicy`, so added the field.